components/brave_wallet_ui/stories/locale.ts

```diff
@@ -203,6 +203,7 @@ provideStrings({
   braveWalletConnectHardwareInfo2: 'Unlock your device and select the $1 app.',
   braveWalletConnectHardwareTrezor: 'Trezor',
   braveWalletConnectHardwareLedger: 'Ledger',
+  braveWalletConnectHardwareAuthorizationNeeded: 'Grant Brave access to your Ledger device.',
```

Comment:
Is this used for only Ledger's or for both? If it's both let's change this to `Grant Brave access to your hardware wallet device.`

Reply:
Thanks, resolving this one.